firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/persistence/CrashlyticsReportPersistence.java

```diff
@@ -270,6 +234,61 @@ private void capFinalizedReports() {
         getAllFilesInDirectory(priorityReportsDirectory), getAllFilesInDirectory(reportsDirectory));
   }
 
+  private File getSessionDirectoryById(String sessionId) {
+    return new File(openSessionsDirectory, sessionId);
+  }
+
+  private void synthesizeReportFile(File sessionDirectory, long sessionEndTime) {
+    final List<File> eventFiles =
+        getFilesInDirectory(sessionDirectory, (f, name) -> name.startsWith(EVENT_FILE_NAME_PREFIX));
```

Comment:
is it worth pulling `(f, name) -> name.startsWith(EVENT_FILE_NAME_PREFIX)` into a static constant?